chaosk8s/pod/actions.py

```diff
@@ -17,19 +15,27 @@
 
 def terminate_pods(label_selector: str = None, name_pattern: str = None,
                    all: bool = False, rand: bool = False,
-                   ns: str = "default", secrets: Secrets = None):
+                   ns: str = "default", secrets: Secrets = None,
+                   mode: str = "fixed", qty: int = 1):
```

Comment:
What other modes could we imagine?

What happens when the provided mode is not supported? Shouldn't we fail at runtime by raising `ActivityFailed`?

Reply:
I see, the default is `fixed` which is the old behavior.

Reply:
Currently, the `mode` can be only `fixed` and `percentage`. But in accordance to [this issue thread](https://github.com/chaostoolkit/chaostoolkit-kubernetes/issues/18#issuecomment-410596445), we can think of supporting range in the future. That's is why mode is a string now. 

Regarding raising an exception, I didn't want to raise an exception when quantity was more than what is present. I was not sure if it is okay to raise an exception only when `mode` is wrongly supplied as I thought it might be inconsistent. What do you think? 
Also, I forgot to handle when negative quantity is given; I feel if we agree to raise an exception for `mode` wrongly supplied, IMO we should raise an exception `when quantity < 0`. (Something like `Any incorrect and incompatible params should be failed`) Your thoughts?


https://github.com/chaostoolkit/chaostoolkit-kubernetes/pull/19#discussion_r212876550
^ yes, default mode is `fixed` with qty `1` I didn't want to change the default functionality for users who are already using it :)

Reply:
>  I was not sure if it is okay to raise an exception only when mode is wrongly supplied as I thought it might be inconsistent.

Ideally we could fail fast during experiment validation but I'm not sure we can at this stage (we could start thinking about it if it was an enum but let's not worry about it now).

I would like we still fail when mode is invalid to help users figuring out why a particular experiment could not operate as expected.

As for the rest, I think you are right indeed.

Reply:
👍  I'll change the code and raise a PR once done.